core/src/main/scala/org/scalarelational/model/SQLDatastore.scala

```diff
@@ -90,7 +90,7 @@ abstract class SQLDatastore protected() extends Datastore {
     b.append(column.name)
     b.append(' ')
     b.append(column.sqlType)
-    if (column.has(NotNull)) {
+    if (!column.isOptional && !column.has(Polymorphic)) {
       b.append(" NOT NULL")
```

Comment:
``column2SQL()`` is used when creating the table, not for the queries. There will be no ``null`` value.